docs/REST API/rest-api.md

```diff
@@ -1082,6 +1089,9 @@ Modifies the Fleet's configuration with the supplied information.
 | email                             | string  | body  | _integrations.zendesk[] settings_. The Zendesk user email to use for this Zendesk integration. |
 | api_token                         | string  | body  | _integrations.zendesk[] settings_. The Zendesk API token to use for this Zendesk integration. |
 | group_id                          | integer | body  | _integrations.zendesk[] settings_. The Zendesk group id to use for this integration. Zendesk tickets will be created in this group. |
+| email                             | string  | body  | _integrations.google_calendar[] settings_. The email address for the Google Workspace service account to be used for this calendar integration. |
+| domain                            | string  | body  | _integrations.google_calendar[] settings_. The domain for the Google Workspace service account to be used for this calendar integration. |
+| private_key                       | string  | body  | _integrations.google_calendar[] settings_. The private key for the Google Workspace service account to be used for this calendar integration. |
```

Comment:
```suggestion
| private_key_json                       | TODO  | body  | _integrations.google_calendar[] settings_. The private key JSON downloaded when generating service account API key to be used for this calendar integration. |
```

Reply:
@rachaelshaw and @getvictor when you get the chance, can you please follow up to the above? Thanks!

Hoping to get these doc changes merged ASAP

Reply:
^^ cc @RachelElysia in case you know the answer (since Victor's OOO)

Reply:
and it can be `null`